hw/ip/flash_ctrl/doc/_index.md

```diff
@@ -222,6 +218,28 @@ The valid program range is thus the valid program resolution for a particular me
 
 This information is not configurable but instead decided at design time and is exposed as a readable status.
 
+#### Erase Suspend
+
+The flash controller supports erase suspend through {{< regref "ERASE_SUSPEND" >}}.
+This allows the software to interrupt an ongoing erase operation.
+
+The behavior of what happens to flash contents when erase is suspended is vendor defined; however, generally it can be assumed that the erase would be incomplete.
+It is then up to the controlling software to take appropriate steps to erase again at a later time.
```

Comment:
Curious: do they implement erase resume, or does erase need to be complete redone?

Reply:
i think it's the latter. 
The suspend i think mostly allows you to safely stop.  So some stuff will be erased..others will not.  However the module does not "remember" where you stopped, as far as i know